CMakeLists.txt

```diff
@@ -36,8 +41,16 @@ if (NOT CMAKE_BUILD_TYPE)
     set(CMAKE_BUILD_TYPE "RelWithDebInfo" CACHE STRING "Choose the type of build, options are: Debug RelWithDebInfo." FORCE)
 endif()
 set(VERSION_TAG "" CACHE STRING "Build date for nightly versioning")
-set(EXTRA_DATA_DIR "" CACHE STRING "Directory for externalized data")
-set(INSTALL_LIB_DIR lib CACHE PATH "Installation directory for libraries")
+if (UNIX AND NOT MINGW AND NOT APPLE)
```

Comment:
you can just check for ${CMAKE_INSTALL_FULL_DATADIR} to make it cleaner.

Reply:
I don't consider that cleaner. Especially not from a readability point of view.